sandbox-web/src/main/java/com/exadel/sandbox/team5/MailSenderRestController.java

```diff
@@ -0,0 +1,21 @@
+package com.exadel.sandbox.team5;
+
+import com.exadel.sandbox.team5.service.MailSenderService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/mail")
+@RequiredArgsConstructor
+public class MailSenderRestController {
```

Comment:
is it needed only for testing?

Reply:
admin can use it for send some message to all users. But i guess it will be redundant when we implement AWS SNS/SQS.

Reply:
ok

Reply:
ok